RELEASING.md

```diff
@@ -37,61 +48,86 @@ Label PRs with the next major version label (e.g. `3.0.0`) and merge changes int
 
 ### Plugin Branching
 
-Plugins are bundled and shiped together along with OpenSearch for every release. Plugin branching follows OpenSearch core branching that will allow working on 3 releases at the same time.
+OpenSearch and OpenSearch Dashboards plugins are bundled and shipped together along with the OpenSearch distribution for every release. Plugin branching follows OpenSearch branching described above that allows working on 3 releases at the same time.
```

Comment:
again, OSD release should decouple from opensearch release in the furture(2025), plugin release should decoupled with OpenSearch and OSD as well. at least OSD

Reply:
Can you share the proposal here?

Reply:
we can update this when this happens